src/dashboard/alerts/alert.py

```diff
@@ -158,10 +158,10 @@ def alert(self):
             emailsystem.send_email_alert(self)
 
         # Log the alert to the log file
-        self.logAlert()
+        self.log_alert()
 
         # Save the alert in the JSON collection for frontend use
-        self.saveAlert()
+        self.save_alert()
```

Comment:
Add `__` to indicate a private function.

Reply:
The __ scrambles the name of the function in python at runtime which has other implications. Python doesn't really have a sense of "private" or "public" methods. I've left them as is in this PR and we can talk more about this during a code review.